core/storage/question/gae_models.py

```diff
@@ -558,6 +558,16 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE
 
+    @classmethod
+    def has_reference_to_user_id(cls, user_id):
+        """Check whether QuestionSummaryModel exist for user.
```

Comment:
maybe `... QuestionSummaryModel has reference to a given user id`?

Reply:
I think that we use this format in the other models.

Reply:
Suggested docstring: "Check whether any existing QuestionSummaryModel refers to the given user_id." -- this seems clearer.

(Similarly for other models.)

Reply:
Done